### ISSUE TITLE
Some Map Images Missing alt text

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#500]
Several of the map images are missing alt text.

Provide the same information that is conveyed in the map as an HTML/text format for an accessible alternative to the image. 

HTML text descriptions with headings and paragraphs would work. Or Data tables. 

Perhaps a standard will need to be giving to those submitting images.

**Comment by sadlerw** [comment]
need clarification on how to proceed with tagging the maps

**Comment by JohnHewittCriterion** [comment]
Recommendation from one of our Auditors:
Write the long description alt text as a visible HTML text with some headings and describe the map fully as if describing all it's parts to someone over the phone. With a good long description a blind person could even be the navigator and direct the driver where to go. 

The Long alt could either be on the page or on a link below that image. 

![samplemap](https://user-images.githubusercontent.com/28604890/37109003-cb111eb6-2206-11e8-985e-dc64a4022da7.png)

Long Description Alt Text:
<h1>Christmas Tree Cutting Area - Canyon Lakes Ranger District</h1>
US Forest Service Department of Agriculture  
Cutting Area open 8:30 am - 4:00 pm. Entrance closes at 2:30 pm.
Visitors must leave the cutting area by 4:00 pm
<h2>Map Key</h2>
Cutting areas are shaded gray with a blue dashed outline.
County Road are a blue and gray dashed line.
Cutting Are Roads are a solid black line.
<h2>Map Description</h2>
Pay Station* is located at intersection of County Road East/West 86/74E and North/South 69. Pay Station has First Aid and Toilets. County Road 74E takes you To Fort Collins. *Note: Smokey Bear, Pay Station and Entrance Station are staffed only on weekends. This map is not to scale.
The Exit (To Pay Station) is on County Road 69 3.25 Miles south of the Pay Station. The Entrance (One Way Only) is located 1.75 miles South East of the Exit on County Road 69. 
Entrance Station* is located 1.60 miles easy of the Entrance on CR 69 and the intersection of CR 68C at Goodell Corner. There is a Chain-up Area, First Aid, and Toilets. CR 69 south 3.20 Miles goes To Rustic and Hwy.
The cutting area is .50 Miles from the Entrance marked 171.A.
Smokey Bear's Lane is the first cutting area road that is one way towards the west and 1.4 miles long.
Smokey Bear* with First Aid and Toilets is at the end of the road at Dasher's Way intersection.
Comet's Lane is a one way cutting area road that goes south west and loops into Dasher's Way after 1.3 Miles.
The intersection of Rudolph's Way is .20 miles north west from Smokey Bear. Rudolph's Way is a one way cutting area road going north east for .85 Miles until it merges into Vixen's Way which his .70 miles to the Exit. 
Dasher's Way is a one way that goes east for .75 miles from the end of Comet's Lane. 
Blitzen's Lane is a one way that forks north east from the start of Dasher's Way and heads toward the exit with 1.5 miles until Vixen's Way. 
Please return this card at the pay station.

**Comment by JaneZC** [comment]
@sadlerw @cameronwolf There are currently 5 maps on the website; two for Mt. Hood and 3 for ARP. If we use the description above, I'll have four descriptions to write.

**Comment by JaneZC** [comment]
@sadlerw   @cameronwolf    and oops! I forgot the 4 on the current starting page; i'd recommend holding off on those until we finish this sprint as we may not have the maps on there by the time we're done.

**Comment by JaneZC** [comment]
@cameronwolf  Here are the five map descriptions in one fell swoop. 

Just a note that I am not 100 percent confident of these descriptions; the maps are not always clear on things like distances to a cutting area, where a cutting area is, etc. etc. I did the best I could with the limited information I had, but I did want to throw a word of caution out there. I'd recommend having someone review it from the forests for accuracy.

[https://app.zenhub.com/files/106473503/6f143ab1-176e-48cf-8c74-3f562095d9f8/download](https://app.zenhub.com/files/106473503/6f143ab1-176e-48cf-8c74-3f562095d9f8/download)[https://app.zenhub.com/files/106473503/e37ec7cc-1742-4696-9b2a-b5eca1fb6e66/download](https://app.zenhub.com/files/106473503/e37ec7cc-1742-4696-9b2a-b5eca1fb6e66/download)[https://app.zenhub.com/files/106473503/645ccd7c-928d-49a2-9552-608ce5f41398/download](https://app.zenhub.com/files/106473503/645ccd7c-928d-49a2-9552-608ce5f41398/download)[https://app.zenhub.com/files/106473503/2827f0a9-c9cd-4dcd-a8af-1cb2a766a12e/download](https://app.zenhub.com/files/106473503/2827f0a9-c9cd-4dcd-a8af-1cb2a766a12e/download)[https://app.zenhub.com/files/106473503/0603b985-ac94-4399-90c2-f87f1cd7305b/download](https://app.zenhub.com/files/106473503/0603b985-ac94-4399-90c2-f87f1cd7305b/download)

**Comment by lauraGgit** [comment]
@sadlerw can you verify that the specs you mentioned in #622 documented in the wiki/readme?

**Comment by sadlerw** [comment]
added the following:

        #Maps and other images

        ##Image size and resolution

        Map and other images that are provided by the forests to be included in the site 
        need to be created at a high enough resolution to be printable by the user. 
        1200 x 1600 should be adequate in most cases.

        ## Image accessibility

        Each image needs to be given a text description for the purposes of section 508 accessibility
        compliance.  In the case of maps, that description should follow the pattern of the current
        map descriptions and be added into config/map-details.json along with a link that includes
        'map description' in the alt text, for example:

        [Sulphur Ranger District map description](/christmas-trees/forests/arp/maps/sulphur)  

        ## Thumbnails

        A thumbnail version of the map may need to be created
        to speed up page load times when not displaying the full-size map.
